Berksfile

```diff
@@ -1,3 +1,5 @@
 source "https://supermarket.getchef.com"
 
 metadata
+
+cookbook 'config-driven-helper', git: 'https://github.com/inviqa/chef-config-driven-helper.git', ref: '1.4.1'
```

Comment:
We only use git as a hack currently in projects for these cookbooks, where its the project that defines the location to the cookbook git repo. The limitations to semver that produces is only really acceptable at a project level as we're already using lock files as well to limit instability. Limiting it to only one bugfix version in cookbooks just adds dependency hell.

We've done the first of my suggestions already in config-driven-helper for a similar reason https://github.com/inviqa/chef-config-driven-helper/blob/master/test/Berksfile

There's nothing stopping us from publishing the cookbook to supermarket, so I think we can go with that.

I was going to go and do this when I start looking at OS'ing hem, which I'm starting to look at today